dbapi_opentracing/tracing.py

```diff
@@ -4,28 +4,66 @@
 import opentracing
 import wrapt
 
+try:
+    from psycopg2.extensions import connection as PsycopgConnection
+    from psycopg2.extensions import cursor as PsycopgCursor
+except ImportError:
+    PsycopgConnection = object
+    PsycopgCursor = object
 
-def _operation_name(caller, function, statement=''):
+
+def _operation_name(caller, func, statement=''):
     """Span operation name obtained from caller's method and sql statement, if any."""
     class_name = caller.__class__.__name__
-    operation_name = function.__name__
+    operation_name = func.__name__
     return '{}.{}({})'.format(class_name, operation_name, statement)
 
 
-class ConnectionTracing(wrapt.ObjectProxy):
-    """A wrapper for a DB API Connection object with traced commit() and rollback() methods."""
+class _ConnectionTracing(object):
+    """
+    Base for traced connections.  Trace flags will be in ObjectProxy attribute format despite not having
```

Comment:
What do you mean by "trace flags" here?

Reply:
Updated language, but I mean the trace/don't trace operation configuration values.